assemble/bin/accumulo-cluster

```diff
@@ -67,15 +67,17 @@ function parse_config {
     exit 1
   fi
 
-  if [[ -z $TSERVER_HOSTS ]]; then
-    echo "ERROR: tservers not found in ${conf}/cluster.yaml"
-    exit 1
-  fi
-
-  for queue in $COMPACTION_QUEUES; do
-    Q="COMPACTOR_HOSTS_${queue}"
+  for group in $COMPACTOR_GROUPS; do
+    Q="COMPACTOR_HOSTS_${group}"
     if [[ -z ${!Q} ]]; then
-      echo "WARN: External compaction queue $queue configured, but no compactors configured for it"
+      echo "WARN: External compaction group $group configured, but no compactors configured for it"
+    fi
+  done
+
+  for group in $TSERVER_GROUPS; do
```

Comment:
It will likely succeed and start everything else, but the root and metadata tables won't be hosted. 1 TabletServer to host the root and metadata tables should be required. Unless we hardcode the group for those tables, it would require checking the config to get the group and then ensuring that at least 1 TabletServer is defined for that group. Ideally 1 Compactor should be required also so that compactions on the root and metadata tables occur.  This was the reason for this [comment](https://github.com/apache/accumulo/issues/3546#issuecomment-1629616298). I think that `cluster.yaml` should look like:

```
manager:
  - localhost
monitor:
  - localhost
gc:
  - localhost
tserver:
  system:
    - localhost
  default:
    - localhost
compactor:
  system:
    - localhost
  default:
    - localhost
sserver:
  default:
    - localhost
```

Not sure if you want to have that discussion here or in #3546

Reply:
>  Unless we hardcode the group for those tables, it would require checking the config to get the group and then ensuring that at least 

We could make the validation a more lax than that and still have something useful.   Could validate that a non empty group of tablet servers exists and a non empty group of compactors exists, without worrying about if the group names or correct.  Then could have a generic message like "There are no tablet servers configured to start, Accumulo requires tablets severs".

Reply:
I added logic in 89e47cd to print a msg and exit if no tservers are found in cluster.yaml.